management-api-server/src/main/java/com/datastax/mgmtapi/resources/common/BaseResources.java

```diff
@@ -65,6 +65,7 @@ protected Response handle(Callable<Response> action) {
           .entity("Internal connection to Cassandra closed")
           .build();
     } catch (Throwable t) {
+      t.printStackTrace();
```

Comment:
Thought:
I put this in to help debug some unexpected 500 responses. May want to convert it to a log? Or simply get rid of it?

Reply:
Why not leave it, if it helps debugging it may as well stay.